perfkitbenchmarker/providers/gcp/flags.py

```diff
@@ -35,3 +35,10 @@
     'https://cloud.google.com/sdk/gcloud/reference/compute/disks/create')
 flags.DEFINE_string('gce_network_name', None, 'The name of an already created '
                     'network to use instead of creating a new one.')
+flags.DEFINE_list('vm_metadata_from_string', [],
+                  'Instance metadata specified through local files. '
+                  'See: --metadata-from-file from '
+                  '`gcloud help compute instances create`.')
+flags.DEFINE_list('vm_metadata', [],
```

Comment:
Done. Prefixed them with gcp_instance, i.e., `gcp_instance_metadata` and `gcp_instance_metadata_from_file`.